projects/jsf-launcher/src/app/schemaV2.json

```diff
@@ -160,14 +160,31 @@
         "dropdowns": {
           "type": "object",
           "name": "Dropdowns",
-          "required": ["dropdown1"],
-          "description": "Dropdowns can be required or not required. If no values are passed in, the form will display \"Select an option...\" by default. If the dropdown is required, then the default option will be hidden once the user has selected a value. If the dropdown is not required, the default option will always be present.",
+          "description": "A value must be selected from a dropdown. A default enum value may be specified",
           "properties": {
             "dropdown1": {
               "display": "dropdown",
               "name": "Dropdown",
               "helpText": "This help text will display when you hover over the information icon",
               "tooltip": "This tooltip will display when you hover over the label.",
+              "enum": [
+                "",
```

Comment:
Food for thought, should the enum also be able to handle a null value?
I know our specific use case might not require this, but there could be a need in the future.

If this is a good chunk of work, it doesn't need to be done for this story. :)